ng2-components/ng2-activiti-form/src/components/widgets/widget.component.ts

```diff
@@ -36,6 +36,13 @@ export class WidgetComponent implements AfterViewInit {
         return this.field ? true : false;
     }
 
+    isRequired(): any {
```

Comment:
This is a special case as I'm binding "required" attribute. This function returns "null" that upon binding removes "required" from html. Otherwise it will be "required"="false" and will still be treated as "required"